providers/class.two-factor-email.php

```diff
@@ -64,11 +64,11 @@ function authentication_page( $user ) {
 			}, 200);
 		</script>
 		<?php
-		submit_button( __( 'Log In', 'two-factor' ) );
+		submit_button( esc_html__( 'Log In', 'two-factor' ) );
```

Comment:
You're escaping this too early.  The string should be passed in to `submit_button()` unescaped, as it will escape it in a contextually correct way when printing.

https://github.com/WordPress/WordPress/blob/c6cff377e7aa9cf566a7511e54a470671cf5088a/wp-admin/includes/template.php#L1956